config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  devise_for :users
+  root to: 'post#index'
+
+  resources :user do
+    resources :post, only: [:new, :create, :show, :edit, :destroy, :update]
```

Comment:
[OPTIONAL] There is no need to use `only:` keyword when you are going to use all methods from resources.